ooniapi/services/ooniprobe/src/ooniprobe/routers/v1/probe_services.py

```diff
@@ -141,4 +174,17 @@ class ProbeUpdateResponse(BaseModel):
 @router.put("/update/{client_id}", tags=["ooniprobe"])
 def probe_update_post(probe_update: ProbeUpdate) -> ProbeUpdateResponse:
     log.info("update successful")
+    
+    # Log update metadata into prometheus
+    probe_update_dict = probe_update.model_dump(exclude_none=True)
+
+    # Info doesn't allows list, if we have a list we have to convert it 
+    # to string
+    if probe_update_dict['supported_tests'] is not None:
+        tests = probe_update_dict['supported_tests']
```

Comment:
I suggest you sort the list before you build the string concatenation to make sure given the same set of tests it's always the same string irrespective of the order in which the probe sent them